src/main/java/org/jenkins/plugins/lockableresources/RequiredResourcesProperty.java

```diff
@@ -160,7 +160,12 @@ public FormValidation doCheckResourceNames(@QueryParameter String value,
 						"Only label, groovy expression, or resources can be defined, not more than one.");
 			} else {
 				List<String> wrongNames = new ArrayList<>();
+				List<String> varNames = new ArrayList<>();
 				for (String name : names.split("\\s+")) {
+					if (name.startsWith("$")) {
```

Comment:
Is this restriction useful? Shouldn't it be possible to expand sub-strings? `printer-${office}` seems like a perfectly valid use case.

Reply:
not allowing `printer-${office}` would be very surprising indeed, excited to test this out !

Reply:
you are right. I'll update this check to only add to `varNames` only if no names were found (i.e. `if (!found)`)